opppy/plot_dump_dictionary.py

```diff
@@ -891,7 +930,9 @@ def setup_parser(self, parser):
         parser.add_argument('-d','--data_name',dest='data_name',help="data to be plotted on the (x,y) contour", type=str)
         parser.add_argument('-x','--x_data', dest='x_value_name', help='dictionary data to be plotted on the x axes.', required=True)
         parser.add_argument('-y','--y_data', dest='y_value_name', help='dictionary data to be plotted on the y axes.', required=True)
-        parser.add_argument('-v','--verts', dest='xy_verts_name', help='dictionary vertices for each cell.', type=str)
+        group = parser.add_argument_group()
+        group.add_argument('-v','--verts', dest='xy_verts_name', help='dictionary vertices for each cell.', type=str)
```

Comment:
If you do not specify the vertices it interpolates the scatter of points to a structured grid (old way of plotting). If you include the verticies it can generate a "mesh" (really just arbitrary polygon patches that are colored based on the data). show_mesh draws the mesh but including a black border on the polygon patches.